modules/database/audit-autoloaded-options/hooks.php

```diff
@@ -26,3 +26,101 @@ function perflab_aao_add_autoloaded_options_test( $tests ) {
 	return $tests;
 }
 add_filter( 'site_status_tests', 'perflab_aao_add_autoloaded_options_test' );
+
+/**
+ * Register custom REST API route for updating autoload.
+ *
+ * @since n.e.x.t
+ */
+function perflab_aao_register_rest_route() {
+	register_rest_route(
+		'perflab-aao/v1',
+		'/update-autoload/(?P<option_name>[a-zA-Z0-9_-]+)',
```

Comment:
Perhaps the option name shouldn't be part of the route, as this could artificially limit which options are able to be manipulated. It's not necessary either, as the option name is being passed among the request body.